src/tools/rbd/action/Import.cc

```diff
@@ -447,6 +511,8 @@ void get_arguments(po::options_description *positional,
   at::add_image_spec_options(positional, options, at::ARGUMENT_MODIFIER_DEST);
   at::add_create_image_options(options, true);
   at::add_no_progress_option(options);
+  options->add_options()
+    ("import-format", po::value<int>(), "format of the file to be imported");
```

Comment:
Yes, this name was changed in the later commit: 2ad0c6ca2f9e0a2a26a91a922b2ec6d9a0dcc190